### ISSUE TITLE
Fix the alt text on the community website

### ISSUE THREAD

**Issue by HarshCasper** [moja-global/community-website#70]
## Description

Currently, most of the images have an `alt` tag but we are still lacking some of the corner cases where we would like the images to be accessible to a larger audience. This issue is part of the larger scope where we are looking to improve the accessibility of our documentation for the general audience.

I am using the [WAVE® Web Accessibility Evaluation Tool](https://wave.webaim.org/) and the associated chrome extension to discover accessibility issues. You can use the same to navigate across the website and discover the images that lack an `alt` tag and document the same.

As a note, the alt-text should be descriptive of the image itself, and should not contain extra information. To understand how to write alt-text, please follow this: https://webaim.org/techniques/alttext/

**Comment by toth2000** [comment]
Looking into this.

**Comment by HarshCasper** [comment]
> Looking into this.

Please let me know if you need any help or assistance on the same. You can alternatively reach out to me on Slack for any questions.

**Comment by toth2000** [comment]
@HarshCasper sure, I will let you know over the community slack channel.

**Comment by Jassi10000** [comment]
@HarshCasper Is the issue unassigned ?
can i work on this  ?

**Comment by sarthakkundra** [comment]
@Jassi10000 You can pick this up if this is still unassigned after you complete the issue assigned to you :)

**Comment by Jassi10000** [comment]
Sure @sarthakkundra

**Comment by toth2000** [comment]
@HarshCasper I have checked the whole site using the mentioned tool but couldn't find a single image without alt text. Maybe we should keep this issue open as new commits are being made to the repo.

**Comment by HarshCasper** [comment]
> @HarshCasper I have checked the whole site using the mentioned tool but couldn't find a single image without alt text. Maybe we should keep this issue open as new commits are being made to the repo.

Sure! I will keep you in an assignment because I and @sarthakkundra are pushing a lot of changes and breaking a lot in the process.
